CONTRIBUTING.md

```diff
@@ -275,8 +274,7 @@ To ensure all your commits are signed, you may choose to add this alias to your
   ```
   [alias]
     amend = commit -s --amend
-    cm = commit -s -m
-    commit = commit -s
+    cm = commit -s
```

Comment:
Why the change here?

Reply:
I added this as an alternative to the original `commit = commit -s` which doesnt work

Reply:
this is not a required change, I have this alias configured and it works.

Reply:
Weird. Okay, I had reverted the change here